apache/superset
# fix: Redirect on 401

### SUMMARY
Redirects the user to the login page when 401s responses are caught. It also passes the `next_url` param which is then used to redirect the user to the original page.

### AFTER

https://user-images.githubusercontent.com/60598000/144454347-c068fd34-7f9c-46d6-b0f8-8d98e6336d1d.mp4


### TESTING INSTRUCTIONS
1. Open any page
2. Delete the cookies or wait for fo the auth to expire
3. Make an action that generates a backend request
4. Observe the behavior

### ADDITIONAL INFORMATION
